packages/backend/src/constants.ts

```diff
@@ -17,5 +17,6 @@
  ***********************************************************************/
 
 // Image related
-export const bootcImageBuilderContainerName = '-bootc-image-builder';
-export const bootcImageBuilderName = 'quay.io/centos-bootc/bootc-image-builder:latest-1714633180';
+export const bootcImageBuilder = 'bootc-image-builder';
+export const bootcImageBuilderCentos = 'quay.io/centos-bootc/bootc-image-builder:latest-1714633180';
+export const bootcImageBuilderRHEL = 'registry.redhat.io/rhel9/bootc-image-builder:9.4';
```

Comment:
There's more than one RHEL version. The 9.4 image is Tech Preview even and isn't going to be updated more and we don't want to use it by default for future RHEL versions like 9.6 or beyond.

I think in general we can just use `registry.redhat.io/rhel$MAJOR/bootc-image-builder:latest` reasonably safely.

To determine `$MAJOR` though requires inspecting the container image; like `podman run <image> bash -c '. /usr/lib/os-release && echo $REDHAT_SUPPORT_PRODUCT_VERSION` or so perhaps.

(In theory we could use `$VERSION_ID` except that may be overridden by downstream builds)